spharpy/spherical.py

```diff
@@ -263,13 +263,21 @@ def renormalize(data, channel_convention, current_norm, target_norm, axis):
         Desired normalization. Valid normalizations are `"n3d"`, `"nm"`
         `"maxN"`, `"sn3d"`, or `"snm"`.
     axis : integer
-        Axis along which the renormalization should be applied
+        Axis along which the renormalization should be applied. The axis
+        contains the spherical harmonics coefficients and must hence have
+        :math:Q = (N+1)^2 channels with :math:N being the spherical harmonics
```

Comment:
This is not typesetting correctly in the docs at the moment. This should work:
```suggestion
        :math:`Q = (N+1)^2` channels with :math:`N` being the spherical harmonics
```